scss/message/_select-mode-mask.scss

```diff
@@ -0,0 +1,12 @@
+.select-mode-mask {
```

Comment:
Is this a "mask" (https://css-tricks.com/clipping-masking-css/) or rather an "overlay"?

Reply:
Sorrt it's an "overlay". Thanks.